Source/Core/DolphinWX/Config/SlippiConfigPane.cpp

```diff
@@ -72,6 +72,10 @@ void SlippiNetplayConfigPane::InitializeGUI()
 	m_slippi_enable_quick_chat_choice->SetToolTip(
 	    _("Enable this to send and receive Quick Chat Messages when online."));
 
+	m_slippi_anonymize_opponents_checkbox = new wxCheckBox(this, wxID_ANY, _("Anonymize Opponents Names"));
+	m_slippi_anonymize_opponents_checkbox->SetToolTip(
+	    _("Replace opponents display names by \'Player [port]\' on Direct and Ranked mode."));
```

Comment:
I think this would only take effect in unranked... and ranked when it's out, but I wouldn't mention ranked for now.

Reply:
right, I'm meant unranked and ranked, specifically not direct. Should I mention ranked now and should I only mention it when you cross that bridge?

Reply:
exclude it for now